### PR TITLE
updates to allow postfix logging

### PR DESCRIPTION
**Pull request by gcstang**

### CODE REVIEW

**Comment by juanluisbaptiste** [comment]
Why not use an external rsyslog container like https://github.com/jumanjihouse/docker-rsyslog ?

**Comment by gcstang** [comment]
Your choice, someone asked for logging so I updated and created a pull request.  Overall simpler to use a single container if that's all you're needing than to configure another one to work with this one but I can see how the docker-rsyslog would be helpful also.  Maybe update your project to utilize one or both ways.

**Comment by juanluisbaptiste** [comment]
I'll take a closer look at both options after the holidays.

**Comment by juanluisbaptiste** [comment]
Hey !

I was for some time looking for another way to enable logging as an alternative to adding a syslog component but got busy and totally forgot, sorry for that. 

I gave it a try tonight but the PR is incomplete, you forgot to modify the Dockerfile to include the new supervisor conf files for rsyslog.

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,19 @@
 #Dockerfile for a Postfix email relay service
 FROM centos:latest
-MAINTAINER Juan Luis Baptiste juan.baptiste@gmail.com
+LABEL maintainer="Juan Luis Baptiste juan.baptiste@gmail.com"
```
Comment (juanluisbaptiste) [inline Dockerfile]: Why this change ? whatever the reason it doesn't belong on this PR.

File: Dockerfile [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
 RUN yum install -y epel-release && yum update -y && \
     yum install -y cyrus-sasl cyrus-sasl-plain cyrus-sasl-md5 mailx \ 
-    perl supervisor postfix
+    perl supervisor postfix rsyslog
 RUN sed -i -e "s/^nodaemon=false/nodaemon=true/" /etc/supervisord.conf
 RUN sed -i -e 's/inet_interfaces = localhost/inet_interfaces = all/g' /etc/postfix/main.cf
+
+COPY etc/supervisord.conf /etc/
```
Comment (juanluisbaptiste) [inline Dockerfile]: Addig the config files can be resumed in one line using wildcards: COPY *.conf /etc

File: Dockerfile (hunk 1 above)
Comment (gcstang) [reply Dockerfile]: It's the proper new way to mark MAINTAINER in docker files.

**Comment by juanluisbaptiste** [comment]
Why did you close it ?

**Comment by gcstang** [comment]
you're correct I obviously didn't test it enough before creating a PR and I don't have time currently but when I do I'll get it updated and create another PR.

My version and this branch is pretty different considering I had to make numerous modifications due to the issue that every time docker restarts my container I end up with another set of echo statements in main.cf because the setup is in run.sh and it's the starting point of the image.  I have to go through what I changed and get that into my own fork for a PR once I get a chance.

**Comment by juanluisbaptiste** [comment]
Well, you could have opened an issue about the duplicate lines too, I have fixed it now and added the rsyslog feature.

**Comment by gcstang** [comment]
Thank you, you're correct.  I could have opened an issue but didn't know your use case and figured it may have worked for you the way it was, I will next time.  I'm glad you figured out what was needed, thank you for your project.
